dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/log/Log4JLogConfigInitializer.java

```diff
@@ -248,6 +254,31 @@ private void configureRootLogger( String filename )
         log.info( "Added root logger using file: " + file );
     }
 
+    private void configureAdditionalLoggers()
```

Comment:
Make call this method `addConfigurableLogs` to be more specific.

Reply:
Sure, just following the convention